pkg/msr/msr_linux.go

```diff
@@ -120,7 +120,7 @@ func (m MSR) Write(c CPUs, data ...uint64) []error {
 }
 
 // MaskBits takes a mask of bits to clear and to set, and applies them to the specified MSR in
-// each of the CPUs.
+// each of the CPUs. Note that MaskBits does not write if the mask does not change the MSR
```

Comment:
I got a bit confused with the name, I did not expect the write. Could we do something like hmm. 
uh.

MaskAndWriteIfChanged

bleah. You call, I'm gonna approve, but the name does confuse me a bit.

Reply:
Changed